src/ngraph/runtime/cpu/pass/cpu_layout.cpp

```diff
@@ -248,6 +249,14 @@ namespace ngraph
 
                     auto arg0_shape = node->get_input_shape(0);
                     auto arg1_shape = node->get_input_shape(1);
+
+                    if (default_weights_format)
+                    {
+                        const size_t IC = 1;
+                        arg0_shape.at(IC) /=
+                            std::dynamic_pointer_cast<ngraph::op::GroupConvolution>(node)
+                                ->get_groups();
+                    }
```

Comment:
Dont we need arg1_shape to be 5D for mkldnn?

Reply:
Not sure. Will it affect the data layout since it's the only one we care about here?
I remember you told me that the data tensor remains 4D, so I did the bare minimum to keep MKLDNN happy and give us the data layout.

Reply:
It might. Can you check with the MKLDNN team on this?